getting-started/stateful-processes.adoc

```diff
@@ -0,0 +1,6 @@
+= Stateful processes using BPMN in 15 minutes 
```

Comment:
The topic can be "Stateful processes using BPMN"
I'm also planning to change the other topic titles that states 15 minutes. Let us not mention it in the title. If required we can mention it in the short description following the title.